app/jenkins-for-jira-ui/src/components/InProductHelpDrawer/InProductHelpDrawer.test.tsx

```diff
@@ -1,13 +0,0 @@
-import React from 'react';
-import { render } from '@testing-library/react';
```

Comment:
Forgot to kill of this file in a previous PR. All drawer tests are in the InProductHelp test file